lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -259,6 +284,62 @@ def _get_rule_match_flow_msgs(self, imsi, _, __, ip_addr, ambr, rule):
             ])
         return msgs
 
+    # This is ng rule installation for stats
+    # pylint: disable=protected-access,unused-argument
+    def _get_ng_rule_match_flow_msgs(self, imsi, _, __, ip_addr, ambr, rule, ng_session_id):
```

Comment:
Why do we have a separate one for 5g, I don't see any significant difference, we can probably reuse